### PR TITLE
Change int to size t for handling very big PHP files

### PR DESCRIPTION
**Pull request by illuusio**
As over 32,767 bytes long PHP file is big it can happen (bundled images or text). Change int to size_t which is unsigned long (4,294,967,295 which is 4GB). After changes it expected that it should not be problem anymore.

### CODE REVIEW

File: blenc.c [hunk 1]
```diff
@@ -412,11 +412,12 @@ static int php_blenc_load_keyhash()
...
 
 
-b_byte *php_blenc_encode(void *script, unsigned char *key, int in_len, int *out_len)
+b_byte *php_blenc_encode(void *script, unsigned char *key, size_t in_len, size_t *out_len)
 {
 	BLOWFISH_CTX *ctx = NULL;
 	unsigned long hi = 0, low = 0;
-	int i, pad_size = 0;
+	size_t i = 0;
+    char pad_size = 0;
```
Comment (Girgias) [inline blenc.c]: Nit: indentation

File: blenc.c [hunk 2]
```diff
@@ -552,15 +553,16 @@ b_byte *php_blenc_decode(void *input, unsigned char *key, int in_len, int *out_l
...
 	PHP_MD5_CTX   context;
 	unsigned char digest[16];
 
 	gettimeofday((struct timeval *) &tv, (struct timezone *) NULL);
-	sec = (int) tv.tv_sec;
-	usec = (int) (tv.tv_usec % 0x100000);
+	sec = tv.tv_sec;
+	usec = (tv.tv_usec % 0x100000);
 
 	spprintf(&tmp, 0, "%08x%05x", sec, usec);
```
Comment (Girgias) [inline blenc.c]: As these conversions really needed when they are just used in the spprintf call below?

**Review by Girgias — APPROVED** [review]
Didn't pull locally, but if the test case failed before the changes and doesn't any more than it should be fine :)

File: blenc.c (hunk 2 above)
Comment (illuusio) [reply blenc.c]: I reverted change

**Pull request merged**